services/core-api/app/api/mines/project_summary/resources/project_summary_list.py

```diff
@@ -116,6 +116,7 @@ def post(self, mine_guid):
             project_summary.save()
             if is_minespace_user():
                 project_summary.send_project_summary_email_to_ministry(mine)
+                project_summary.send_project_summary_email_to_proponent(mine)
         except Exception as e:
```

Comment:
We should only send the email when it goes from draft -> open. Can you add that additional logic for both emails?